modules/tinymce/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - New optional `defaultExpandedIds` and `onToggleExpand` options to the `tree` component config. #TINY-9653
+- New `accordion` plugin with the `InsertAccordion` command, toolbar, and menu bar button to insert an accordion. #TINY-9730
```

Comment:
Maybe make it less verbose. Hardly to expect we can describe all those things added with the new plugin in one change log entry.
```suggestion
- New `accordion` plugin with the `InsertAccordion` command. #TINY-9730
```

Reply:
Thanks, shortened the CHANGELOG entry!